engines/dfc_provider/app/services/supplied_product_builder.rb

```diff
@@ -13,16 +13,25 @@ def self.supplied_product(variant)
       description: variant.description,
       productType: product_type(variant),
       quantity: QuantitativeValueBuilder.quantity(variant),
+      spree_product_uri: id,
```

Comment:
On further thought, this doesn't work. One use case is to group several variants (supplied products) because they belong to the same spree product. A client application can't group the variants if they all have their own id value. Instead, they need a common value like the product id to be grouped.